<launch>
  
  <!--
    Leds 1
  -->
  <node pkg="nao_behavior_tree" type="nao_leds" name="nao_leds1" output="screen">
    <rosparam command="load" file="$(find nao_behavior_tree)/launch/parameters.yaml"/>
    <param name="NAO_IP" value="192.168.1.19"/>
    <param name="NAO_PORT" value="9559"/>
  </node>
  
  <!--
    Leds 2
  -->
  <node pkg="nao_behavior_tree" type="nao_leds" name="nao_leds2" output="screen">
    <rosparam command="load" file="$(find nao_behavior_tree)/launch/parameters.yaml"/>
    <param name="NAO_IP" value="192.168.1.127"/>
    <param name="NAO_PORT" value="9559"/>
  </node>
  
  
  <node pkg="nao_behavior_tree" type="Obstacles" name="Obstacles" args="1" output="screen">
  	<param name="mode" value="0"/>
    <param name="cameraId" value="1"/>
    <param name="webcam" value="true"/>
  </node>
  
</launch>
